integration_tests/features/BaseNodeAutoUpdate.feature

```diff
@@ -1,14 +1,14 @@
 @auto_update
 Feature: AutoUpdate
 
-
     # Not sure why this takes so long on CI
-    @long-running
-    Scenario: Auto update finds a new update
+    @long-running @broken
```

Comment:
Cool, I think let's fix this in another PR

Reply:
Yeah, it's not the highest priority. Since the update is just info.